test/tools/yulopti.cpp

```diff
@@ -265,10 +411,11 @@ int main(int argc, char** argv)
 		po::options_description options(
 			R"(yulopti, yul optimizer exploration tool.
 	Usage: yulopti [Options] <file>
-	Reads <file> as yul code and applies optimizer steps to it,
+	Reads <file> containing a yul object and applies optimizer steps to it,
```

Comment:
Why this change? What happens if no object is present in the file?

Reply:
The PR enables that objects (and nested objects) can be read by the tool. But you can have just a raw yul block too.
If there is no object nor yul block present in the file, then it terminates with an exception due to parsing error.
@dflupu , I think that it should say that a raw yul block is also accepted.